install.md

```diff
@@ -0,0 +1,30 @@
+# Paranomax Mobile App
+
+## Installation steps for Windows
+
+### Development environment
+Choose one of the environments below to write/maintain the code:
+- [Git](https://git-scm.com/downloads)
+- [GitHub CLI](https://cli.github.com/)
```

Comment:
I don't think github cli is mandatory to maintain the code, is it?

Reply:
True is a listing of different possibilities.